frontend/src/entry/model/settings/EntryRolesInfo.tsx

```diff
@@ -54,36 +54,105 @@ export default function EntryRolesInfo({ entry }: EntryRolesInfoProps) {
+                    <TableRow>
+                      <TableCell>Permission</TableCell>
+                      <TableCell align='right'>Owner</TableCell>
+                      <TableCell align='right'>Contributor</TableCell>
+                      <TableCell align='right'>Consumer</TableCell>
+                      <TableCell align='right'>Release Reviewer</TableCell>
+                      <TableCell align='right'>Access Reviewer</TableCell>
+                    </TableRow>
+                  </TableHead>
+                  <TableBody>
+                    {rows.map((row) => (
+                      <TableRow key={row.permission} sx={{ '&:last-child td, &:last-child th': { border: 0 } }}>
+                        <TableCell component='th' scope='row'>
+                          {row.permission}
+                        </TableCell>
+                        <TableCell align='center'>{row.roles.includes('owner') ? <Done /> : <></>}</TableCell>
+                        <TableCell align='center'>{row.roles.includes('contributor') ? <Done /> : <></>}</TableCell>
+                        <TableCell align='center'>{row.roles.includes('consumer') ? <Done /> : <></>}</TableCell>
+                        <TableCell align='center'>{row.roles.includes('releaseReviewer') ? <Done /> : <></>}</TableCell>
+                        <TableCell align='center'>{row.roles.includes('accessReviewer') ? <Done /> : <></>}</TableCell>
+                      </TableRow>
+                    ))}
+                  </TableBody>
+                </Table>
+              </TableContainer>
+            </Stack>
+            <Stack spacing={1}>
+              <Box>
+                <Typography component='h3' variant='h6' fontWeight='bold'>
+                  Dynamic Roles
+                </Typography>
+                <Typography variant='caption'>
+                  {`The following roles are specified by this ${toSentenceCase(entry.kind)}'s schema.`}
+                </Typography>
+              </Box>
+              {schemaRolesList}
+            </Stack>
+          </Stack>
         </Stack>
       )}
     </>
   )
 }
+
+const rows = [
+  {
+    permission: 'View private model',
+    roles: ['owner', 'contributor', 'consumer', 'releaseReviewer', 'accessReviewer'],
+  },
+  {
+    permission: 'Edit a model card',
+    roles: ['owner', 'contributor'],
+  },
+  {
+    permission: 'Draft a release',
+    roles: ['owner', 'contributor'],
+  },
+  {
+    permission: 'Edit a release',
+    roles: ['owner', 'contributor'],
+  },
+  {
+    permission: 'Request inferencing service',
+    roles: ['owner', 'contributor'],
+  },
+  {
+    permission: 'Push an image',
+    roles: ['owner', 'contributor'],
+  },
+  {
+    permission: 'Edit or delete an inferencing form',
+    roles: ['owner', 'contributor'],
+  },
+  {
+    permission: 'Request access',
+    roles: ['owner', 'contributor', 'consumer'],
+  },
+  {
+    permission: 'Edit or delete any access request',
```

Comment:
This is a bit of an awkward one because you can also edit or delete an access request if you are named in the additional contacts of said access request. Maybe we could use a `HelpPopover` in this permission cell to explain that?

Something like this perhaps?
"You can also edit/delete an access request if you are an Additional Contact"